test/ModuleLoader.test.cpp

```diff
@@ -1,18 +1,47 @@
+#include "GameModules/IpDropDown.h"
 #include "ModuleLoader.h"
 #include <gtest/gtest.h>
+#include <memory>
 
-TEST(ModuleLoader, InitializeNoModules)
+
+TEST(ModuleLoader, NoModulesLoaded)
+{
+	ModuleLoader moduleManager;
+	EXPECT_EQ(0, moduleManager.Count());
+	EXPECT_EQ("", moduleManager.GetModuleName(0));
```

Comment:
Should this be "" or an exception? For the exported C interface, there are no exceptions, so it made sense to return "" there instead.

Reply:
Yeah, I think this should probably be an exception. It's trying to access data about a non-existent module. There should be no reason for that last line to ever be executed. Especially since the first line allows code to guard against it, and more likely, loop mechanics would prevent it from happening, because really, a method like that is likely only going to be called from some kind of loop.

Reply:
OK, done